packages/zmarkdown/config/latex/prepare-quizz.js

```diff
@@ -0,0 +1,22 @@
+const visit = require('unist-util-visit')
+
+module.exports = processQuizzFactory
+const clone = require('clone')
+function processQuizzFactory (ctx) {
+  const correctionTitle = ctx.correctionTitle || 'Correction'
+  return function quizzCustomBlockVisitor (node, index, parent) {
+    node.type = 'neutralCustomBlock'
+    const correction = clone(node)
+    correction.type = 'sCustomBlock'
+    correction.children[0].children[0].value = correctionTitle
+    parent.children.splice(index + 1, 0, correction)
+    const bodyChildren = node.children[1].children
```

Comment:
Same here: what happens if `node.children[1]` doesn't exist? Please add at least a small check.

Reply:
This one, I will make the test but once more, we configured custom block parsing so that it forces to have a title and a block